.gitignore

```diff
@@ -1,5 +1,6 @@
 bin
 Configuration.Override.props
+Version.props
```

Comment:
This change should no longer be necessary, as this PR now generates `bin\Build$(Configuration)\Version.props`.

Reply:
done